docs/memoir/2025.07.20.md

```
@@ -0,0 +1,17 @@
+## 2025.07.20 - 공백기 이후 기능 구현 방향성 조정
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)